### PR TITLE
Minor updates

### LINKED ISSUE #9: grapes-or-or-grapes CMD check
Will need to pass to get on CRAN:

prepare_Rd: grapes-or-or-grapes.Rd: \name should not contain !, | or @

### PR DESCRIPTION
**Pull request by wch**
The first commit fixes #9.

The second commit causes pkgsnap to not try to restore packages with an unknown source.  Previously, calling `restore()` when some packages have an unknown source results in output like this when it tries to download the package:

```
  Unknown package source: ... ERROR.
```

After it downloads all the packages, it gives errors like this when it tries to read the DESCRIPTION file for the missing packages:

```R
Error in file(con, "rb") : cannot open the connection
In addition: Warning messages:
1: In FUN(X[[i]], ...) : Unknown package source: 
2: In FUN(X[[i]], ...) : Unknown package source: 
3: In FUN(X[[i]], ...) : Unknown package source: 
4: In FUN(X[[i]], ...) : Unknown package source: 
5: Cannot download package bubbles 
6: Cannot download package rsconnect 
7: Cannot download package shiny 
8: Cannot download package shinySignals 
9: In file(con, "rb") : cannot open file 'NA': No such file or directory
```

### CODE REVIEW

**Comment by dougmet** [comment]
Thanks! @gaborcsardi, are you OK with us handling this or do you still want to maintain this package?

**Comment by gaborcsardi** [comment]
@dougmet Do you want to handle it? :)

File: R/restore.R [hunk 1]
```diff
@@ -30,6 +30,9 @@ restore <- function(from = "packages.csv", R = TRUE, ...) {
   # Remove this package (pkgsnap) from the list
   pkgs <- pkgs[pkgs$Package!="pkgsnap", ]
 
+  # Don't try to install packages that have an unknown source
+  pkgs <- pkgs[!is.na(pkgs$Source), ]
+
```
Comment (gaborcsardi) [inline R/restore.R]: Maybe this is worth a warning?

Comment (wch) [reply R/restore.R]: Good idea.

**Comment by dougmet** [comment]
@gaborcsardi ha ha, happy to, but don't want to step on toes.

**Pull request merged**